walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -1,27 +1,88 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" dir="ltr">
   <head>
-    <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <meta charset="utf-8">
     <link rel="stylesheet" href="style.css">
+    <title>Raymond's Portfolio</title>
   </head>
   <body>
-    <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+    <div class="header-container">
+        <a class="header-item" href="#home">Raymond C. Alvarez Velez</a>
```

Comment:
Indentation should use 2 spaces here and below.

Reply:
done